app/src/main/java/com/google/android/stardroid/activities/dialogs/TimeTravelDialog.java

```diff
@@ -244,6 +244,9 @@ private void setPopularDate(int popularDateIndex) {
       case 8:  // Moon Landing 1969.
         setDate(new Date(-14182953622L));
         break;
+      case 9:
+        setDate(new Date(1712534400));
```

Comment:
Is this date correct? It has many fewer digits than the others.

Reply:
Well @jaydeetay I used the time stamp date converter as the source. I myself don't know how to calculate these date and times for be honest!